### DISCUSSION TITLE
regional text like hindi ,marathi language is not supported in PDF export for BIRT report

### DISCUSSION THREAD

**Discussion by ravitejabollu** [eclipse-birt/birt#1284] [Q&A]
regional text like hindi ,marathi is not supported in PDF export for BIRT report,

**Comment by speckyspooky** [comment]
There are some discussion articles with the same content.
Please take a look there because additional configuration is necessary to gte the full support of embedded fonts.

Two discussion examples:
https://github.com/eclipse/birt/discussions/1236
https://github.com/eclipse/birt/discussions/898
